src/compensations/Compensations.tsx

```diff
@@ -52,9 +57,27 @@ const Compensations = ({ compensations }: CompensationsProps) => {
     );
   };
 
+  const updateLocation = (id: string) => {
+    setSelectedLocation(id);
+  };
+
+  const locationOptions: IOption[] = Object.values(locations).map(
```

Comment:
Since `locations` should be a list, we shouldn't use `Object.values(...)` here

Reply:
Good catch!